changelog/9.2.1.yaml

```diff
@@ -0,0 +1,31 @@
+version: 9.2.1
+entries:
+    - kind: bug-fix
+      summary: Fix issue prevent checkin local_metadata from being updated
+      description: |
+        Once an Elastic Agent is marked with audit unenrolled Fleet Server fails to update the document in
+        Elasticsearch when it checks in. This fixes that issue an now the Fleet Server will be able to update the
+        document in Elasticsearch reflecting the actual status of the Elastic Agent.
```

Comment:
```suggestion
        After an Elastic Agent was marked with audit unenrolled, Fleet Server failed to update the document in
        Elasticsearch when it checked in. This fixes that issue, and now the Fleet Server can update the
        document in Elasticsearch, reflecting the actual status of the Elastic Agent.
```

Reply:
Fixed typo and added punctuation for clarity. 
Also reworked issue to be past tense and kept fix in present tense. 

@ebeahan, please keep me honest that I didn't change the meaning